tools/constants.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * @file tools/constants.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2003-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class constants
+ * @ingroup tools
+ *
+ * @brief Get the value of application constants.
+ */
+
+require(dirname(dirname(dirname(dirname(__FILE__)))) . '/tools/bootstrap.inc.php');
+
+class constants extends CommandLineTool {
```

Comment:
Heh, it's a bad sign that we need this...

Reply:
It's been very helpful! :)